.gitignore

```diff
@@ -11,3 +11,5 @@ media/
 static-files/
 static/vendor/
 tmp/
+package-lock.json
+src/
```

Comment:
@jhdulaney Huh. Neither of these show up in my local setup. I'm guessing `package-lock.json` is related to the npm setup, but I'm curious what's creating this – do you happen to know?

Reply:
Yes, they show up during npm setup.